### DISCUSSION TITLE
How to speed up import time

### DISCUSSION THREAD

**Discussion by kukuhprastyono** [DefectDojo/django-DefectDojo#10763] [Q&A]
I run the Defectdojo (2.26.1) via docker-compose and use mysql-redis profile, on my cicd pipeline i run Trivy image scan and import the report.json file to Defectdojo via [api/v2/reimport-scan/], this step takes >= 3 minutes to complete when import new product/service. 
this things causes timeout error and then pipeline error
any suggestion to speed up Defectdojo import time?

**Comment by mtesauro** [comment]
Just for FYI: There's generally more responses and back and forth in the #defectdojo channel on OWASP's Slack - details to join at:
https://github.com/DefectDojo/django-DefectDojo?tab=readme-ov-file#community-getting-involved-and-updates
